src/EditorFeatures/Core/NavigationBar/NavigationBarController.cs

```diff
@@ -92,11 +101,12 @@ internal partial class NavigationBarController : IDisposable
         _visibilityTracker = visibilityTracker;
         _uiThreadOperationExecutor = uiThreadOperationExecutor;
         _asyncListener = asyncListener;
+        _nonFrozenComputationCancellationSeries = new(threadingContext.DisposalToken);
```

Comment:
We don't have a token for when a file is closed?

Reply:
you are correct.  and i *keep* making that mistake!

Reply:
(basically, i often see the IThreadingContext and just say in my 'this has the same lifetime as that').